lib/elixir/lib/enum.ex

```diff
@@ -932,7 +932,8 @@ defmodule Enum do
 
   @doc """
   Returns a new enumerable appending the result of invoking `fun` on
-  each corresponding item of `enumerable`.
+  each corresponding item of `enumerable` and flattening
+  the given results (only one level deep)..
```

Comment:
Sounds perfect! This is exactly what I expect to see there. Let's change these two flat_maps :D